packages/storybook/.storybook/preview-body.html

```diff
@@ -0,0 +1,2 @@
+<!-- Reliable element for i18next mutation observer -->
+<main lang="en"></main>
```

Comment:
I'm not sure that we want this because our actual story doesn't go inside of it:

![empty `<main>` element](https://user-images.githubusercontent.com/2008881/165363240-a0384715-6041-4e65-b8c6-83eea2351962.png)

Reply:
This is what removes the flakiness observed in `Docs` pages. Without this, sometimes Storybook doesn't attach the i18next event listener because it doesn't find a main element.